### PR TITLE
18867: Local Residency- Children under 19 need to show an "Attested" status 

### PR DESCRIPTION
**Pull request by saipraveen18**
|Field|Value|
|-----|-----|
| Redmine Ticket Number | [18867](https://devops.dchbx.org/redmine/issues/18867) |
| App-Dev Road Map # | n/a |
| Start Date | n/a |
| Due Date | n/a  |
| App-Dev Priority | UP-1 |
| Development Story Points | n/a |
| Peer Review Story Points | 0.5 |
| Ticket Author | @karashearer5 |

### CODE REVIEW

File: app/helpers/verification_helper.rb [hunk 1]
```diff
@@ -16,6 +16,7 @@ def doc_status_label(doc)
   def verification_type_status(type, member, admin=false)
     consumer = member.consumer_role
     return "curam" if (consumer.vlp_authority == "curam" && consumer.fully_verified? && admin)
+    return 'attested' if (type == 'Residency' && member.age_on(TimeKeeper.date_of_record) <= 18)
```
Comment (saikumar9) [inline app/helpers/verification_helper.rb]: image

File: spec/helpers/verification_helper_spec.rb [hunk 2]
```diff
@@ -76,10 +76,24 @@
...
+
+      it 'does not return attested if age > 18 and type is residency' do
+        person.dob = Date.new(1988,11,10)
+        person.consumer_role.update_attributes!(local_residency_validation: 'valid')
+        expect(helper.verification_type_status('Residency', person)).not_to eq('attested')
+      end
+
+      it 'does not return attested if age <= 18 and type is social security number ' do
+        expect(helper.verification_type_status('Social Security Number', person)).not_to eq('attested')
+      end
```
Comment (saikumar9) [inline spec/helpers/verification_helper_spec.rb]: Good Spec coverage.

File: app/helpers/verification_helper.rb (hunk 1 above)
Comment (saikumar9) [reply app/helpers/verification_helper.rb]: Shows  "Attested" Status in the Documents page for children under 19 and type "Residency".

Comment (Kevin-Wei) [reply app/helpers/verification_helper.rb]: 👍

**Pull request merged**